### PR TITLE
Add export functionality to cli

### PR DESCRIPTION
**Pull request by Fortiz2305**

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -27,20 +35,37 @@ def records_count():
...
         'prep-stat': insert_line_by_line_prepared_statements,
-        'batch': batch_insert
-
+        'batch': batch_insert,
     }
+
     if mode == 'copy':
         insert_copy(logger)
+    elif mode == 'copy-events':
+        insert_copy_events(logger=logger, csv_path='/opt/data_events.csv')
```
Comment (gardenunez) [inline cli.py:50]: this could be in the modes dict.

File: cli.py [hunk 2]
```diff
@@ -27,20 +35,37 @@ def records_count():
...
+@click.option('--mode', type=click.Choice(['line-by-line', 'line-by-line-json', 'copy']),
+              default='line-by-line')
+def export(mode):
+    modes = {
+        'line-by-line': export_line_by_line,
+        'line-by-line-json': export_line_by_line_json,
+    }
+
+    if mode == 'copy':
+        export_copy(logger=logger, csv_path='/opt/test.csv')
```
Comment (gardenunez) [inline cli.py:65]: this can be in the modes dict

**Review by gardenunez — APPROVED** [review]

File: cli.py (hunk 1 above)
Comment (Fortiz2305) [reply cli.py:50]: But it needs another csv file

Comment (gardenunez) [reply cli.py:50]: true

**Pull request merged**